src/Microsoft.AspNetCore.Razor.VSCode/package.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "microsoft.aspnetcore.razor.vscode",
-  "version": "0.0.1",
+  "defaults": {
```

Comment:
You don't need to change this package.json do you? I'd figure you'd just need to change `https://github.com/aspnet/Razor.VSCode/blob/master/client/package.json`

Reply:
Done. Can we merge it now ?